blocks/library/gallery/index.js

```diff
@@ -119,10 +121,16 @@ registerBlockType( 'core/gallery', {
 		}
 
 		return (
-			<div className={ `blocks-gallery align${ align }` }>
+			<div className={ `blocks-gallery align${ align } columns-${ columns }` }>
 				{ images.map( ( img, i ) => (
 					<GalleryImage key={ i } img={ img } />
```

Comment:
Doesn't look like you introduced this, but maybe we should consider using `img` here for the key instead of the list index, as the list index can lead to bugs when the images end up reordered. The key will be the same but the images different and React will optimize away the change.